CargoMonitor/CargoMonitor.cs

```diff
@@ -265,7 +265,6 @@ private void _handleCargoEvent(CargoEvent @event)
                             if (total != cargo.total || stolen != cargo.stolen || missionCount != cargo.haulageData.Count())
                             {
                                 UpdateCargoFromInfo(cargo, cargoInfo);
-                                if (@event.update) { return; }
```

Comment:
I think the change makes sense. 
Correct me if I'm wrong, but isn't the `update` parameter in `@event.update` vestigial after this change?
If so, maybe we should remove the parameter altogether?

Reply:
You're not entirely wrong. It was used for that purpose, but it's also let's you know where the cargo info came from, startup or json. I'm reticent to remove that as we may want/need it in the future.

Reply:
Ok then. :-)